README.md

```diff
@@ -57,12 +57,14 @@ sudo make install
 
 Frontends
 ===
+  - [Weasel](https://github.com/rime/weasel): frontend for Windows
+  - [PIME](https://github.com/EasyIME/PIME): another Rime frontend for Windows
+  - [Squirrel](https://github.com/rime/squirrel): frontend for macOS
+  - [XIME](https://github.com/stackia/XIME): yet another Rime frontend for macOS
+  - [Trime](https://github.com/osfans/trime): frontend for Android
+  - [iRime](https://itunes.apple.com/cn/app/irime-shu-ru-fa/id1142623977?mt=8): frontend for iOS(**source code can't find**)
```

Comment:
If I doubt whether it's a valid open-source project, please exclude it.

The purpose of this README is not to have a complete list of related work, but a reference to open-source projects based on Rime for *developers* to know where to go if they are interested in contributing.